Mathlib/FieldTheory/KrullTopology.lean

```diff
@@ -275,3 +275,15 @@ theorem krullTopology_totallyDisconnected {K L : Type*} [Field K] [Field L] [Alg
 #align krull_topology_totally_disconnected krullTopology_totallyDisconnected
 
 end TotallyDisconnected
+
+lemma IntermediateField.fixingSubgroup_top (K L : Type*) [Field K] [Field L] [Algebra K L] :
```

Comment:
I think this can be a simp-lemma? Can you please also prove the "dual": fixing `bot` gives `top`.